src/Helpers/VulnerabilitiesHelper.js

```diff
@@ -113,7 +113,7 @@ export function createCveListByAccount({ cveList, parameters }) {
                     parent: index * 2
                 }
             ]);
-        return { data, meta, isLoading, errors };
+        return { data, meta: { ...meta, cvesCount }, isLoading, errors };
```

Comment:
you don't need `cvesCount`. The response returns  `meta: total_items`

Reply:
I am afraid, we can not use `meta: total_items`. As it is total items count not on last the page but on the whole table.

Reply:
I see, you want the length of the items for a particular request. makes sense now